sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -157,108 +128,82 @@ export class EventPosition {
-}
-
-// @public
-export class InMemoryPartitionManager implements PartitionManager {
-    claimOwnership(partitionOwnership: PartitionOwnership[]): Promise<PartitionOwnership[]>;
-    listOwnership(fullyQualifiedNamespace: string, eventHubName: string, consumerGroupName: string): Promise<PartitionOwnership[]>;
-    updateCheckpoint(checkpoint: Checkpoint): Promise<string>;
-}
-
-// @public
-export interface LastEnqueuedEventInfo {
-    enqueuedTime?: Date;
+export interface LastEnqueuedEventProperties {
+    enqueuedOn?: Date;
     offset?: string;
-    retrievalTime?: Date;
+    retrievedOn?: Date;
     sequenceNumber?: number;
 }
 
 export { MessagingError }
 
 // @public
-export interface ParentSpanOptions {
-    parentSpan?: Span | SpanContext;
+export interface OperationOptions {
+    abortSignal?: AbortSignalLike;
+    tracingOptions?: {
+        spanOptions?: SpanOptions;
+    };
 }
 
 // @public
-export interface PartitionCheckpointer {
+export interface PartitionContext {
+    readonly consumerGroup: string;
+    readonly eventHubName: string;
+    readonly fullyQualifiedNamespace: string;
+    readonly lastEnqueuedEventProperties?: LastEnqueuedEventProperties;
+    readonly partitionId: string;
     updateCheckpoint(eventData: ReceivedEventData): Promise<void>;
-    updateCheckpoint(sequenceNumber: number, offset: number): Promise<void>;
-    // @internal (undocumented)
-    updateCheckpoint(eventDataOrSequenceNumber: ReceivedEventData | number, offset?: number): Promise<void>;
 }
 
 // @public
-export interface PartitionContext {
-    consumerGroupName: string;
+export interface PartitionOwnership {
+    consumerGroup: string;
+    etag?: string;
     eventHubName: string;
     fullyQualifiedNamespace: string;
-    partitionId: string;
-}
-
-// @public
-export interface PartitionManager {
-    claimOwnership(partitionOwnership: PartitionOwnership[]): Promise<PartitionOwnership[]>;
-    listOwnership(fullyQualifiedNamespace: string, eventHubName: string, consumerGroupName: string): Promise<PartitionOwnership[]>;
-    updateCheckpoint(checkpoint: Checkpoint): Promise<string>;
-}
-
-// @public
-export interface PartitionOwnership extends PartitionContext {
-    eTag?: string;
-    lastModifiedTimeInMS?: number;
-    offset?: number;
+    lastModifiedTimeInMs?: number;
     ownerId: string;
-    ownerLevel: number;
-    sequenceNumber?: number;
+    partitionId: string;
 }
 
 // @public
 export interface PartitionProperties {
     beginningSequenceNumber: number;
     eventHubName: string;
     lastEnqueuedOffset: number;
+    lastEnqueuedOnUtc: Date;
     lastEnqueuedSequenceNumber: number;
-    lastEnqueuedTimeUtc: Date;
     partitionId: string;
 }
 
 // @public
-export type ProcessCloseHandler = (reason: CloseReason, context: PartitionContext & PartitionCheckpointer) => Promise<void>;
+export type ProcessCloseHandler = (reason: CloseReason, context: PartitionContext) => Promise<void>;
```

Comment:
Any chance of the word "Process" here is confused with node processes, or the `process` variable in general or any other kind of "process"?
Do we even need the prefix? If we do, then how about "Partition"?
`PartitionCloseHandler`, `PartitionEventHandler`, `PartitionErrorHandler` and `PartitionInitializationHandler` because all 4 are about what happens when we are processing a partition?